<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicGenerSettings</class>
 <widget class="QDialog" name="BasicGenerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Generator Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/icons/sbicon.png</normaloff>:/icons/sbicon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: cyan;
}

QRadioButton:disabled {
	color: gray;
}
QRadioButton::indicator:disabled {
	border-image: url(:/radio/radio_disabled.png);
}
QRadioButton::indicator {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off.png);
}
QRadioButton::indicator:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off_hover.png);
}
QRadioButton::indicator:pressed {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_pressed.png);
}
QRadioButton::indicator:checked {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected.png);
}
QRadioButton::indicator:checked:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected_hover.png);
}

QDoubleSpinBox {
	color: cyan;
	border-image: url(:/spinbox/spinbox_frame.png);
}
QDoubleSpinBox:hover {
	border-image: url(:/spinbox/spinbox_frame_hover.png);
}
QDoubleSpinBox:focus {
	border-image: url(:/spinbox/spinbox_frame_edit.png);
}
QDoubleSpinBox::up-button {
	height: 11px;
	width: 20px;
	border-image: url(:/spinbox/spinbox_upper_idle.png);
}
QDoubleSpinBox::down-button {
	height: 11px;
	width: 20px;
	border-image: url(:/spinbox/spinbox_lower_idle.png);
}
QDoubleSpinBox::up-button:hover {
	border-image: url(:/spinbox/spinbox_upper_hover.png);
}
QDoubleSpinBox::up-button:pressed {
	border-image: url(:/spinbox/spinbox_upper_pressed.png);
}
QDoubleSpinBox::down-button:hover {
	border-image: url(:/spinbox/spinbox_lower_hover.png);
}
QDoubleSpinBox::down-button:pressed {
	border-image: url(:/spinbox/spinbox_lower_pressed.png);
}

QPushButton {
	border-image: url(:/buttons/button75_idle.png);
}
QPushButton:disabled {
	color: gray;
	border-image: url(:/buttons/button75_disabled.png);
}
QPushButton:hover {
	border-image: url(:/buttons/button75_hover.png);
}
QPushButton:pressed {
	border-image: url(:/buttons/button75_pressed.png);
}

QComboBox {
	border-image: url(:/combobox/combobox_frame_idle.png);
}
QComboBox:hover {
	border-image: url(:/combobox/combobox_frame_hover.png);
}
QComboBox:disabled {
	border-image: url(:/combobox/combobox_frame_disabled.png);
	color: gray;
}
QComboBox::drop-down {
	border-image: url(:/combobox/combobox_button_idle.png);
}
QComboBox::drop-down:hover {
	border-image: url(:/combobox/combobox_button_hover.png);
}
QComboBox::drop-down:pressed {
	border-image: url(:/combobox/combobox_button_pressed.png);
}
QComboBox::drop-down:disabled {
	border-image: url(:/combobox/combobox_button_disabled.png);
}
QComboBox QAbstractItemView {
	background-color: black;
}


</string>
  </property>
  <widget class="QDoubleSpinBox" name="ampBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>35</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets this (and only this) generator's volume.</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="ampLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="phaseLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>55</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="phaseBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sets the wave's starting point.</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.083333000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="freqBiasLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Note Bias</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="noteBiasBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>85</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Biases the generator's frequency by the number halfsteps here.</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	color: cyan;
	border-image: url(:/spinbox/spinbox_frame.png);
}
QSpinBox:hover {
	border-image: url(:/spinbox/spinbox_frame_hover.png);
}
QSpinBox:focus {
	border-image: url(:/spinbox/spinbox_frame_edit.png);
}
QSpinBox::up-button {
	height: 11px;
	width: 20px;
	border-image: url(:/spinbox/spinbox_upper_idle.png);
}
QSpinBox::down-button {
	height: 11px;
	width: 20px;
	border-image: url(:/spinbox/spinbox_lower_idle.png);
}
QSpinBox::up-button:hover {
	border-image: url(:/spinbox/spinbox_upper_hover.png);
}
QSpinBox::up-button:pressed {
	border-image: url(:/spinbox/spinbox_upper_pressed.png);
}
QSpinBox::down-button:hover {
	border-image: url(:/spinbox/spinbox_lower_hover.png);
}
QSpinBox::down-button:pressed {
	border-image: url(:/spinbox/spinbox_lower_pressed.png);
}</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QGraphicsView" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>185</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView {
	border-image: url(:/popups/basicgener_backdrop.png);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="attackLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>105</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="releaseLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="attackBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The attack time in seconds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="releaseBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>135</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The release time in seconds.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="waveTypeBox">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>60</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Sine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Triangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Square</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sawtooth</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oval</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Peak</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="waveTypeLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wave Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>ampBox</zorder>
  <zorder>ampLabel</zorder>
  <zorder>phaseLabel</zorder>
  <zorder>phaseBox</zorder>
  <zorder>closeButton</zorder>
  <zorder>applyButton</zorder>
  <zorder>freqBiasLabel</zorder>
  <zorder>noteBiasBox</zorder>
  <zorder>attackLabel</zorder>
  <zorder>releaseLabel</zorder>
  <zorder>attackBox</zorder>
  <zorder>releaseBox</zorder>
  <zorder>waveTypeBox</zorder>
  <zorder>waveTypeLabel</zorder>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
